### PR TITLE
fix: jsonld parse is ignored when context url include the trailing slash

### PR DESCRIPTION
**Pull request by tianzhich**
Some articles use the jsonld in which their `@context` is `http://schema.org/`. It will be ignored by the current regular expression test. Then we can't get the correct metadata.

This PR fix this issue.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ node_modules
 .sw?
 *.jar
 *.xpi
+.vscode
```
Comment (fchasen) [inline .gitignore:11]:
```suggestion
```
I would recommend adding this to your `~/.gitignore` instead.

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,9 +54,7 @@ reasonable).
 
 ### Added
 
-- `isProbablyReaderable` [can now take an optional options object](https://github.com/mozilla/readability/pull/634) to configure it,
-allowing you to specify the minimum content length, minimum score, and how to
-check if nodes are visible.
+- `isProbablyReaderable` [can now take an optional options object](https://github.com/mozilla/readability/pull/634) to configure it, allowing you to specify the minimum content length, minimum score, and how to check if nodes are visible.
```
Comment (fchasen) [inline CHANGELOG.md:57]: Can you revert this? We'd prefer the line to wrap.

**Review by fchasen — CHANGES_REQUESTED** [review]
Thanks for these changes and the test, just some small nits to fix.

**Comment by tianzhich** [comment]
> Thanks for these changes and the test, just some small nits to fix.

Hi Fred, Thanks for your time. Now it would be ok.

**Review by fchasen — APPROVED** [review]

**Pull request merged**